js/apps/admin-ui/src/identity-providers/add/SamlGeneralSettings.tsx

```diff
@@ -27,15 +27,15 @@ export const SamlGeneralSettings = ({
 
   const {
     register,
-    watch,
+    control,
     formState: { errors },
   } = useFormContext();
 
-  const alias = watch("alias");
+  const alias = useWatch({control, name: "alias"});
 
   return (
     <>
-      <RedirectUrl id={id} />
```

Comment:
Since `id` is no longer used it can be removed from the props for this component.